site/docs/components/icon/examples.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Icon
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+  <LivePreview componentName="icon" exampleName="BasicIcon">
+    ### Basic Icon
+
+    Icons should be individually imported and used from the `@salt-ds/icons`
+    package
+
+  </LivePreview>
+  <LivePreview componentName="icon" exampleName="Size">
+    ### Icon Size
+
+    You can change the size of each icon using the `size` property which acts as a multiplier for the base icon’s size, as described in the [Scale Foundation](/salt/foundations/scale). The base size of Icon is based on the active density as described in the [Size Foundation](/salt/foundations/size).
+
+  </LivePreview>
+  <LivePreview componentName="icon" exampleName="SolidIcons">
+    ### Solid Icons
+
+    We provide two types of icons: default and solid. Most icons will have both versions, with some exceptions as described in the [Iconography Foundation](/salt/foundations/iconography). The outline style is the default icon type. Use the solid icon for additional emphasis or to represent a toggled 'on' state.
+
+  </LivePreview>
+  <LivePreview componentName="icon" exampleName="Color">
+    ### Icon Color
+
+    To override the default color of the icon, use the --saltIcon-color CSS variable. This could be necessary to match primary or secondary text. You can find appropriate color variables in the [Typography Foundation](/salt/foundations/typography#color)
```

Comment:
```suggestion
    To override the default color of the icon, use the `--saltIcon-color` CSS variable. This could be necessary to match primary or secondary text. You can find appropriate color variables in the [Typography Foundation](/salt/foundations/typography#color)
```